index.html

```diff
@@ -6,213 +6,237 @@
     <title>Document</title>
     <link rel="stylesheet" href="index.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>    
 <body>
-    <section class="f-section">
-        <div class="box">
-            <img id="s" src="hero/M1.png"  alt="snapshot" height="250px auto" width=350px auto>
-        <h2 class="title">Tonic</h2>
-        <ul class="subheading">   
+    <main>
+
+    <div class="All-section">    
+    <section class="first">
+        <header>
+            
+            <span><p class="logo">My Logo  </p></span>
+            
+                <ul class="menu">
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                </ul>
+                <a href="" class="more"><img src="hero/humberger.png" height="25px"  alt=""></a>
```

Comment:
@Sohail-Aslam i would suggest to check the break point of your desktop version menu: Since it is not showing expected: